piperider_cli/workspace.py

```diff
@@ -907,6 +911,33 @@ def generate_report(input=None):
     console.print(f"Report generated in {dir}/index.html")
 
 
+def _append_descriptions_from_dbt(profile_result, dbt, default_schema):
+    dbt_root = os.path.expanduser(dbt.get('projectDir'))
+    full_cmd_arr = ['dbt', 'list', '--output', 'json', '--resource-type', 'all', '--output-keys', 'resource_type,description,name,columns,source_name']
```

Comment:
we could use `dbt list --output json --resource-type model --resource-type source` to select two resources 

Example:
```json
dbt list --output json --resource-type model --resource-type source
{"resource_type": "model", "depends_on": {"macros": [], "nodes": ["source.infuse_finance.PUBLIC.PRICE"]}, "config": {"enabled": true, "alias": null, "schema": null, "database": null, "tags": [], "meta": {}, "materialized": "table", "persist_docs": {}, "quoting": {}, "column_types": {}, "full_refresh": null, "unique_key": null, "on_schema_change": "ignore", "post-hook": [], "pre-hook": []}, "unique_id": "model.infuse_finance.PRICE_PRESENT", "package_name": "infuse_finance", "original_file_path": "models/PRICE_PRESENT.sql", "name": "PRICE_PRESENT", "alias": "PRICE_PRESENT", "tags": []}
{"unique_id": "source.infuse_finance.PUBLIC.ACTION", "package_name": "infuse_finance", "original_file_path": "models/schema.yml", "name": "ACTION", "source_name": "PUBLIC", "resource_type": "source", "tags": [], "config": {"enabled": true}}
{"unique_id": "source.infuse_finance.PUBLIC.PRICE", "package_name": "infuse_finance", "original_file_path": "models/schema.yml", "name": "PRICE", "source_name": "PUBLIC", "resource_type": "source", "tags": [], "config": {"enabled": true}}
{"unique_id": "source.infuse_finance.PUBLIC.SYMBOL", "package_name": "infuse_finance", "original_file_path": "models/schema.yml", "name": "SYMBOL", "source_name": "PUBLIC", "resource_type": "source", "tags": [], "config": {"enabled": true}}
```

Reply:
GOOD POINT!
modified